packages/relay-service/src/index.js

```diff
@@ -39,6 +45,33 @@ const sendMessage = async ({keyPairDoc, recipientDid, message, type}) => {
     keyAgreementKey,
   });
 
+  if (useDIDServiceEndpoint) {
```

Comment:
i would suggest that if a did document has a service endpoint, it should always be used for didcomm instead of a relay (below this check)

Reply:
sure that makes sense. We have the wallet to wallet verification that uses didcomm messages but it is using did:key only, so i guess it should be safe

Reply:
I've pushed a commit for this update